wpilib/wpilib/robotdrive.py

```diff
@@ -86,6 +89,7 @@ def __init__(self, *args, **kwargs):
         numbers are passed, the motorController keyword argument, if present,
         is the motor controller class to use; if unspecified, :class:`.Talon` is used.
         """
+        warnings.warn("Deprecated in 2018.0.0. Use DifferentialDrive, KilloughDrive, or MecanumDrive instead.", DeprecationWarning, stacklevel=2)
```

Comment:
The user still can't really tell what exactly has been deprecated from this message.

Reply:
Do you think it should be more like "RobotDrive was deprecated in 2018.0.0"?